code/modules/power/singularity/singularity.dm

```diff
@@ -283,11 +283,14 @@
 	//set background = BACKGROUND_ENABLED
 	var/ngrabbed=0
 	for(var/atom/X in orange(grav_pull, src))
+		if(istype(X,/atom/movable/lighting_overlay))//god that's disgusting

```

Comment:
I think you can get away with a normal type check rather than a more expensive istype check seeing as lighting_overlay doesn't have any children and shouldn't